WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -33,6 +33,12 @@ extension UserDefaults {
         return value(forKey: key.rawValue) as? T
     }
 
+    /// Returns the boolean value (if any) associated with the specified Key.
+    ///
+    func bool(forKey key: Key) -> Bool? {
+        return bool(forKey: key.rawValue)
+    }
+
```

Comment:
You can delete this. With the power of [generics](https://docs.swift.org/swift-book/LanguageGuide/Generics.html), it's already been defined above on Line 32. Pretty nifty!

Reply:
Ah right! Thanks!!

Fixed in 9a2690f366374cef604a6083f828b689fb299502